app/core/settings.py

```diff
@@ -100,6 +101,10 @@
 
 STATIC_URL = '/static/'
 
+CONSUMER_GROUPS_HEADER = 'HTTP_X_CONSUMER_GROUPS'
+KONG_AUTH_HEADER = 'Digests-Can-Preview'
```

Comment:
for consistency this could be `KONG_PREVIEW_HEADER`, but thinking about it this is part of the side that is agnostic to what the api-gateway is, so it can just be `AUTHORIZATION_PREVIEW_HEADER` (with `AUTHORIZATION_MODIFICATION_HEADER` following)